source/documentation/rdo-packaging-guidelines.html.md

```diff
@@ -26,8 +26,10 @@ some adjustments may be needed for each particular case.
 
 ## Packages Review Process
 
-* All reviews **must** block the review tracker using the alias RDO-**release** .
-Announcing reviews on the RDO mailing list is encouraged to raise awareness within the
+* All reviews **must** add the alias RDO-{**release**} to the "Blocks" field on the review request BZ.
+Each release has a BZ created aliased RDO-{**release**}. For example, for the R release add RDO-ROCKY to the Blocks field on the review request BZ.
```

Comment:
Can you add a link to https://bugzilla.redhat.com/show_bug.cgi?id=RDO-ROCKY on RDO-ROCKY ?

Reply:
yup will do